mysql/tests/test_statements.py

```diff
@@ -525,6 +525,47 @@ def run_query(q):
     assert metric['dd_commands'] == expected_metadata_payload['commands']
 
 
+@pytest.mark.parametrize("disable_sql_obfuscation", [True, False])
+def test_deobfuscated_statement(aggregator, dd_run_check, dbm_instance, datadog_agent, disable_sql_obfuscation):
+    dbm_instance['obfuscator_options'] = {'disable_sql_obfuscation': disable_sql_obfuscation}
+    mysql_check = MySql(common.CHECK_NAME, {}, [dbm_instance])
+
+    test_query = "SELECT table_schema FROM information_schema.tables WHERE table_schema = 'processlist'"
+    query_signature = 'b75cb4aaad0c138c'
+    normalized_query = 'SELECT table_schema FROM information_schema.tables WHERE table_schema = ?'
+
+    def _obfuscate_sql(query, options=None):
+        if 'WHERE table_schema' in query:
+            return re.sub(r'\s+', ' ', normalized_query or '').strip()
+        return re.sub(r'\s+', ' ', query or '').strip()
```

Comment:
This seems a little hacky. One option is to make this an e2e test so we can use the obfuscator.

If we do want it to be a unit test, we could just prepend: `obfuscated: ` or something to the query and assert on that. That way it is a bit less mysterious when debugging.

Reply:
> One option is to make this an e2e test so we can use the obfuscator.

TIL

Thanks for the suggestions: 9e848e1b8165ba5bb472146e5a34817bde9b5c84